dashboard/app/models/course.rb

```diff
@@ -51,7 +51,7 @@ def self.load_from_path(path)
     hash = JSON.parse(serialization)
     course = Course.find_or_create_by!(name: hash['name'])
     course.update_scripts(hash['script_names'])
-    course.update!(properties: hash['properties'])
+    course.update!(teacher_resources: hash.try(:[], 'properties').try(:[], 'teacher_resources'))
```

Comment:
The worry is that we add a future property, we expect it to be deserialized from the file, and it isn't because we're only loading the teacher_resources prop.